### DISCUSSION TITLE
How to invoke an Edge Function using the `fetch` API?

### DISCUSSION THREAD

**Discussion by TimDev9492** [supabase/supabase#12526] [Questions]
I want to invoke an Edge Function using the fetch API for more control. What headers do I need to set in a browser (and what are the required values) to get the same "level" of authentication that I'd get using this:
```js
const supabase = createBrowserSupabaseClient();

supabase.functions.invoke('my-function');
```
Any help is much appreciated.

**Comment by GaryAustin1** [comment]
https://supabase.com/docs/guides/functions/quickstart#invoking-remote-functions
The authorization header would need to be anon or a non-expired user jwt.
